### DISCUSSION TITLE
Show plotting percentage in CLI

### DISCUSSION THREAD

**Discussion by ntrixter** [Chia-Network/chia-blockchain#2839] [Q&A] [answered]
Is there a way to view what the plotting progress percentage is when plotting using the CLI?  I can see what bucket it is working on but I would like to know total progress like shown in the GUI.

**Comment by Forusim** [comment, marked as answer]
As of now you have to parse the log for major steps described here: https://github.com/Chia-Network/chia-blockchain/wiki/Beginners-Guide#create-a-plot
